### DISCUSSION TITLE
Contributing to `pyvista/vtk-data`

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [pyvista/pyvista#1789] [Q&A] [answered]
If I want to create a pull request to add a dataset to `pyvista/vtk-data`, should I fork the repository? It contains lfs files and downloading counts against the maintainer's bandwidth.

Are there details on contributing to `vtk-data`? The `Contributing.md` only talks about licenses (i.e. don't add a data set for which there isn't a license).

**Comment by banesullivan** [comment, marked as answer]
@adam-grant-hendry, perhaps the best way to do this is to create an issue in `vtk-data` with a link to the data you'd like to include. That way, someone with enough bandwidth can upload the data.

Further, I may look into removing any git-lfs files/move them to a separate repo

**Reply by adam-grant-hendry** [reply]
Thanks @banesullivan. FYI, if you're interested, I created a PR to add a DICOM stack for testing an improvement I made to `add_volume()` and incorporate a DICOM directory reader. Your thoughts are welcome! ([Add `DICOM stack` data set #5](https://github.com/pyvista/vtk-data/pull/5))
